components/Ink/src/private/MDCLegacyInkLayer.m

```diff
@@ -612,7 +635,17 @@ - (void)evaporateToPoint:(CGPoint)point completion:(void (^)(void))completionBlo
   [self resetBottomInk:YES toPoint:point completion:completionBlock];
 }
 
-#pragma mark - MDCLegacyInkLayerRippleDelegate
+#pragma mark - MDCLegacyRippleInkLayerDelegate
+
+- (void)animationDidStart:(CAAnimation *)anim {
+  if (!self.isAnimating) {
```

Comment:
The `isAnimating` state (as most of the state) is not optimal. But we have prevent sending multiple callbacks as we have e.g. foreground and background legacy ink layer ripple that callback here.